src/qt/coincontroldialog.cpp

```diff
@@ -741,6 +742,10 @@ void CoinControlDialog::updateView()
             // address
             CTxDestination outputAddress;
             QString sAddress = "";
+            if(!treeMode && nIsMint){
```

Comment:
We should not mix normal UTXO and Mints together by default. The available coins to show should depend on what type of transaction:

- Normal send: show only normal UTXO.
- Mint: show both normal UTXO and Mint but not able to mix it together.
- Spend: show only mints.

Reply:
Wouldn't it be better to only show normal UTXO for Mint?

Reply:
That is what I think before but after discuss with @thebevrishot users might want to merge their smaller denomination to the bigger one (e.g. input evevel `0.1` to create `1`).

Reply:
When minting why do you want to show mint for selection? Mint for mint?

Reply:
@reubenyap please see my above comment.

Reply:
I think this is a rare case but sure. Go ahead if the fix is easy.